com.unity.multiplayer.mlapi/package.json

```diff
@@ -1,16 +1,18 @@
 {
-  "name": "com.unity.multiplayer.mlapi",
-  "displayName": "MLAPI Networking Library",
-  "version": "0.1.0",
-  "unity": "2019.4",
-  "unityRelease": "8f1",
-  "description": "This the Core Unity Mid-level API that provides core SDK for multiplayer games within unity",
-  "keywords": [
-    "unity"
-  ],
-  "type": "library",
-  "hideInEditor": false,
-  "dependencies": {
-    "com.unity.nuget.mono-cecil": "1.10.1-preview.1"
-  }
+    "name": "com.unity.multiplayer.mlapi",
```

Comment:
I think this rewrite is simply caused by spaces and things. I'm not changing anything other than the 2 new dependencies below, FYI.